test/hga/HGA_Regression.ipynb

```diff
@@ -166,16 +196,15 @@
    "outputs": [],
    "source": [
     "if environment_information is not None:\n",
-    "    granules = {Environment.PROD: 'G1817336128-ASF',\n",
-    "                Environment.UAT: 'G1244144599-EEDTEST',\n",
-    "                Environment.SIT: 'G1244144599-EEDTEST'}\n",
+    "    granule_id = get_test_granule_id(environment_information['env'], 'G1817336128-ASF',\n",
```

Comment:
This makes it look like you could use a prod environment. So maybe a prod environment should be added above, and let it error when it tries to get the collection information on L146?  Well, I guess the test never gets there and it does just say "Skipping test: HGA regression tests not configured for this environment." so maybe that's fine.

Reply:
I'm leaning towards we should alert the user if they are trying to run this in a way that is unconfigured and maybe all of these tests should raise an error rather than just printing a message. 🤔

Reply:
Yeah - my concern here is when Bamboo automatically runs the tests for a production release, we don't want to just fail. Hopefully, once we've had that meeting in August, we can better identify the tests we need, along with the corresponding production collections. (I'm not 100% confident the production ASF collections will work as configured)